bitnami/fluentd/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd
-version: 2.0.2
+version: 2.0.3
```

Comment:
Please bump the minor version as it is adding a feature

Reply:
Done :-)